packages/widgetbook/README.md

```diff
@@ -1,23 +1,44 @@
 <img height=40 src="https://raw.githubusercontent.com/widgetbook/widgetbook/2107e1afe2217e8ecde56c6ade1fd3706c3e6570/docs/assets/WidgetbookLogo.svg">
 
-[![Discord](https://img.shields.io/discord/879618555560218625?color=blue&style=flat-square)](https://discord.com/invite/zT4AMStAJA)
+[![Discord](https://img.shields.io/discord/879618555560218625?color=blue&style=flat-square&logo=discord)](https://discord.com/invite/zT4AMStAJA)
 [![style: very good analysis](https://img.shields.io/badge/style-very_good_analysis-B22C89.svg?style=flat-square)](https://pub.dev/packages/very_good_analysis) 
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?style=flat-square)
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?label=test&style=flat-square)
-
-Widgetbook is a flutter package which helps developers cataloguing their widgets, testing them quickly on multiple devices and themes, and sharing them easily with designers and clients. Inspired by Storybook.js.
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/widgetbook/widgetbook/ci.yaml?branch=main&style=flat-square&)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/widgetbook/widgetbook/ci.yaml?label=test&style=flat-square)
 
+Widgetbook is a Storybook for Flutter which helps developers cataloguing their widgets, testing them quickly on multiple devices and themes, and sharing them easily with designers and clients. Inspired by Storybook.js.
 
+<p align="center">
+<img src="https://raw.githubusercontent.com/widgetbook/widgetbook/main/docs/assets/Screenshot.jpg" height="400" alt="Widgetbook Screenshot" />
+</p>
 
+# Features
 
+- 💙 **Made for Flutter**. Easy to use. 
+- 🦄 **Open source**.
+- 🍭 **Feature rich**. Test your Widgets on different devices, themes, locales, text scales, etc.
+- ⚙️ **Customize for your needs**. Easily add additional settings to your liking. 
+- 🔧 **Maintained** by a fast growing Flutter startup from Germany.
+- 👩🏼‍💻 **Support** via our [Discord](https://discord.com/invite/zT4AMStAJA) community.
 
+Widgetbook can do much more
 
+- 📙 **Catalogue existing Widgets**. Never develop Widgets twice. 
+- 📦 **Develop in isolation**. Don't think about business logic. 
+- 🎛 **Knobs**. Define how your team interacts with your Widgets by defining customizable knobs. 
+- ⚡️ **Hot reload**. Quickly find and fix bugs.
+- ✨ **Beautiful documentation**. Readable, easy to understand and ever-improving.
```

Comment:
Do you think it makes sense to add a link to the documentation here?

Reply:
Thought about it. Most people will look at the readme via pub.dev which has a documentation button in the sidebar. Do you think it's necessary to put a link here?

Reply:
Hmm I see your point. Still a link in this context might be helpful sometimes in my opinion if there is no harm in adding it.

Reply:
Yup. Added it.